op-node/node/node.go

```diff
@@ -309,71 +304,6 @@ func (n *OpNode) initRuntimeConfig(ctx context.Context, cfg *Config) error {
 	return nil
 }
 
-func (n *OpNode) initL1BeaconAPI(ctx context.Context, cfg *Config) error {
```

Comment:
why is this deleted, instead of using BSC alternative method?

Reply:
yes, we don't need this